From ac24d562f490cbbca0280c2b5502e5502acdacc3 Mon Sep 17 00:00:00 2001
From: Imre Deak <imre.deak@intel.com>
Date: Wed, 1 Jul 2026 18:31:39 +0300
Subject: [PATCH 03400/15333] drm/i915/dp_link_caps: Drop noupdate postfix from
 max link limit set helpers

There is no need to update any maximum link information when updating
the max link limits, so drop the related noupdate postfix from the
helpers setting the max link limits.

Reviewed-by: Suraj Kandpal <suraj.kandpal@intel.com>
Signed-off-by: Imre Deak <imre.deak@intel.com>
Link: https://patch.msgid.link/20260701153204.4124150-11-imre.deak@intel.com
---
 drivers/gpu/drm/i915/display/intel_dp_link_caps.c | 15 +++++++--------
 1 file changed, 7 insertions(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/i915/display/intel_dp_link_caps.c b/drivers/gpu/drm/i915/display/intel_dp_link_caps.c
index de520bd423d5..3ef678d47c71 100644
--- a/drivers/gpu/drm/i915/display/intel_dp_link_caps.c
+++ b/drivers/gpu/drm/i915/display/intel_dp_link_caps.c
@@ -545,26 +545,26 @@ bool intel_dp_link_caps_filter_add(struct intel_dp_link_caps *link_caps,
 	return true;
 }
 
-static void set_max_link_limits_no_update(struct intel_dp_link_caps *link_caps,
-					  const struct intel_dp_link_config *max_link_limits)
+static void set_max_link_limits(struct intel_dp_link_caps *link_caps,
+				const struct intel_dp_link_config *max_link_limits)
 {
 	link_caps->max_limits = *max_link_limits;
 }
 
-static void reset_max_link_limits_no_update(struct intel_dp_link_caps *link_caps)
+static void reset_max_link_limits(struct intel_dp_link_caps *link_caps)
 {
 	struct intel_dp_link_config max_link_limits = {
 		.rate = intel_dp_max_common_rate(link_caps),
 		.lane_count = intel_dp_link_caps_max_common_lane_count(link_caps),
 	};
 
-	set_max_link_limits_no_update(link_caps, &max_link_limits);
+	set_max_link_limits(link_caps, &max_link_limits);
 }
 
 static void reset_max_link_limits_reenable_all(struct intel_dp_link_caps *link_caps)
 {
 	link_caps->enabled_configs = INTEL_DP_LINK_CAPS_FILTER_ALL;
-	reset_max_link_limits_no_update(link_caps);
+	reset_max_link_limits(link_caps);
 }
 
 /**
@@ -640,7 +640,7 @@ bool intel_dp_link_caps_set_max_limits(struct intel_dp_link_caps *link_caps,
 	if (!max_link_limits_valid(link_caps, max_link_limits))
 		return false;
 
-	set_max_link_limits_no_update(link_caps, max_link_limits);
+	set_max_link_limits(link_caps, max_link_limits);
 
 	return true;
 }
@@ -654,7 +654,7 @@ bool intel_dp_link_caps_set_max_limits(struct intel_dp_link_caps *link_caps,
  */
 void intel_dp_link_caps_reset_max_limits(struct intel_dp_link_caps *link_caps)
 {
-	reset_max_link_limits_no_update(link_caps);
+	reset_max_link_limits(link_caps);
 }
 
 static int intel_dp_link_config_bw(struct intel_dp_link_caps *link_caps,
@@ -860,7 +860,6 @@ int intel_dp_link_config_index(struct intel_dp_link_caps *link_caps,
  */
 void intel_dp_link_caps_reset(struct intel_dp_link_caps *link_caps)
 {
-	/* TODO: Update the maximum link information. */
 	reset_max_link_limits_reenable_all(link_caps);
 }
 
-- 
2.55.0

